roles/azure_manage_postgresql/defaults/main.yml

```diff
@@ -1,3 +1,4 @@
 ---
 azure_manage_postgresql_operation: create
 azure_manage_postgresql_postgresql_version: "11"
+azure_manage_postgresql_postgresql_settings: []
```

Comment:
I think instead of adding empty default we should handle the "not defined" case in the role itself.
It should be done by adding the following lines:
```
  when:
    - azure_manage_postgresql_postgresql_settings is defined
    - azure_manage_postgresql_postgresql_settings | length > 0
```
What do you think @p3ck @prabinovRedhat ?
This is something we should change in multiple places, so I'm ok with fixing it here now or leaving it with the default solution and opening new task for this

Reply:
I can add it , I'll update the PR

Reply:
The PR is updated, @nirarg @p3ck